snp_motion_planning/src/planning_server.cpp

```diff
@@ -553,11 +554,51 @@ class PlanningServer
     tesseract_planning::TaskComposerFuture::UPtr result = executor->run(*task, problem, task_data);
     result->wait();
 
-    // Save the output dot graph
+    // Save the output motion planning data
```

Comment:
I think this is complex enough now that it would make sense to push this into it's own function